implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs

```diff
@@ -23,6 +23,8 @@ use crate::config::lookup::InternetAddress;
 
 use crate::{fmt_warn, ConnectionStatus};
 
+pub const DEFAULT_NODE_NAME: &str = "_default_node_name";
```

Comment:
I don't think this will work as expected. This value is used in command as a temporary value that is replaced by either a random name or the name defined in a node configuration. The only reliable way to get the default node name would be to read it from the DB.